### ISSUE TITLE
The `CustomValidationPopup` should have a property to prevent opening on Keyboard focus

### ISSUE THREAD

**Issue by timunie** [MahApps/MahApps.Metro#4086]
## Describe the feature
Sometimes the UI gets disturbed by opening the Popups frequently. An `AttachedProperty` like `ShowValidationErrorOnMouseOver` for the `HasFocus` property could be useful here. The Popup can still be shown on `MouseOver`

## Additional context
Idea: 
```c#
ShowValidationErrorOnKeyboardFocus= true | false ;  // The default value would be true like the current behavior
```

I can implement this when the other things are implemented. I don't want to spam too many PR at a time. 

Happy coding
Tim

**Comment by punker76** [comment]
@timunie Any chance to implement this now?
